### PR TITLE
QGB CLI takes time to execute

### PR DESCRIPTION
**Pull request by SweeXordious**
The CLI commands were taking time to execute because a testing helper file was initializing a lot of resources for the tests, and the file wasn't suffixed with `_test`.  Thus, it was initializing those resources on every command run.

Closes https://github.com/celestiaorg/celestia-app/issues/451

### CODE REVIEW

**Comment by SweeXordious** [comment]
depends on https://github.com/celestiaorg/celestia-app/pull/524

**Comment by evan-forbes** [comment]
great find!!

File: testutil/common.go [hunk 1]
```diff
@@ -1,10 +1,11 @@
-package keeper
+package testutil
```
Comment (evan-forbes) [inline testutil/common.go:1]: :+1:

**Review by evan-forbes — APPROVED** [review]
:+1:

File: x/qgb/orchestrator/orchestrator.go [hunk 2]
```diff
@@ -183,6 +184,11 @@ func (orch Orchestrator) EnqueueMissingEvents(
 
 	orch.Logger.Info("syncing missing nonces", "latest_nonce", latestNonce, "last_unbonding_height", lastUnbondingHeight)
 
+	// To accommodate the delay that might happen between starting the two go routines above.
+	// Probably, it would be a good idea to further refactor the orchestrator to the relayer style
+	// as it is entirely synchronous. Probably, enqueing separatly old nonces and new ones, is not
+	// the best design.
+	// TODO decide on this later
```
[the remark this answers is not in the document]
Comment (SweeXordious) [reply x/qgb/orchestrator/orchestrator.go:191]: It's alright, we were exploring possibilities. I will open an issue as a nice to have.

[the remark this answers is not in the document]
Comment (SweeXordious) [reply x/qgb/orchestrator/orchestrator.go:191]: https://github.com/celestiaorg/celestia-app/issues/546

**Pull request merged**